packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.tsx

```diff
@@ -101,6 +103,7 @@ function useDateRangeCalendarDefaultizedProps<TDate>(
     minDate: applyDefaultDate(utils, themeProps.minDate, defaultDates.minDate),
     maxDate: applyDefaultDate(utils, themeProps.maxDate, defaultDates.maxDate),
     calendars: themeProps.calendars ?? 2,
+    disableDragEditing: themeProps.disableDragEditing ?? false,
```

Comment:
You can add this prop to `DateRangeCalendarDefaultizedProps`

Reply:
Could you expand on this point?
It's added to `DateRangeCalendarProps` and is hence propagated to `DateRangeCalendarDefaultizedProps` and initialized here in `useDateRangeCalendarDefaultizedProps` 🤔

Reply:
Unless I missread, `DateRangeCalendarDefaultizedProps['disableDragEditing']` has a type of `boolean | undefined` when in fact it is `boolean`.

We just have to add it to the list of defaultized props in this type (like `calendars` for instance)

It's clearly a detail, but it will avoid having future developers add a `props.disableDragEditing!` before passing it to a function that expects a boolean for instance.

I had several of those hacks for validation props in the past before cleaning the typing.

Reply:
Thanks for explaining. Will do. 😉